File: responses/ENS7D8-DXV/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    # write your code in Python
    alist = list(range(1,x))
    #print(alist)
    blist = []
    for i in alist :
        if (i%3 == 0 or i%5 == 0) :
            if(i%3 == 0 and i%5 == 0) :
                pass
            else:
                blist.append(i)
    #print(blist, sum(blist))
    return sum(blist)
    

File: responses/MKQGAM-AWS/reverse_digits.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(x) {
    var sum=0;
   for(var i=0;i<x;i++){
       if(i%3==0 && i%5==0){
       }
       else if(i%3==0){
           sum+=i;
       }
       else if(i%5==0){
           sum+=i;
       }
   }
   return sum;
}